spec/text_spec.rb

```diff
@@ -0,0 +1,40 @@
+require_relative "spec_helper"
+
+describe "PDF::Core::Text" do
+  before do
+    class TextMock
+      include PDF::Core::Text
+
+      attr_reader :text
+
+      def add_content(str)
+        @text ||= ""
+        @text << str
+      end
+    end
+    @mock = TextMock.new
+  end
```

Comment:
It probably would be better to move class definition out of `before` and use `let` instead to create a new mock for each example.